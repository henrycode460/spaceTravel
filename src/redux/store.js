import { configureStore } from '@reduxjs/toolkit';
import { combineReducers } from 'redux';
import missionsReducer from './missions/missions';
import rockertsReducer from './rockets/rockets';

const rootStore = combineReducers({
   missions: missionsReducer,
   rockets: rockertsReducer,
});

const store = configureStore({
  reducer: rootStore,
});

export default store;
