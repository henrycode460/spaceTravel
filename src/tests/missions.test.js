import renderer from 'react-test-renderer';
import Missions from "../components/missions";
import { Provider } from 'react-redux';
import store from '../redux/store.js'

it('Testing missions component:', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Missions />
    </Provider>).toJSON();
  expect(tree).toMatchSnapshot();
});