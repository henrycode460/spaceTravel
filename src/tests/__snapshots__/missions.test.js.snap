// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing missions component: 1`] = `
<div>
  <hr />
  <table
    className="table"
  >
    <thead>
      <tr>
        <th>
          Mission
        </th>
        <th>
          Description
        </th>
        <th>
          Status
        </th>
        <th>
          empty
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
